packages/leancode_lint/test/test_cases/add_cubit_suffix_for_cubits_test.dart

```diff
@@ -21,31 +22,28 @@ class AddCubitSuffixForYourCubitsTest extends AnalysisRuleTest {
   }
 
   Future<void> test_main() async {
-    await assertDiagnostics(
-      '''
+    await assertDiagnosticsInRanges('''
 import 'package:flutter_bloc/flutter_bloc.dart';
 
 class ProperlyNamedCubit extends Cubit<bool> {
   ProperlyNamedCubit() : super(true);
 }
 
-class NotProperlyNamed extends Cubit<bool> {
+class /*[0*/NotProperlyNamed/*0]*/ extends Cubit<bool> {
   NotProperlyNamed() : super(true);
 }
 
-class NotProperlyNamedWithDifferentSuperClass extends ProperlyNamedCubit {
+class /*[1*/NotProperlyNamedWithDifferentSuperClass/*1]*/ extends ProperlyNamedCubit {
   NotProperlyNamedWithDifferentSuperClass();
 }
 
 class ProperlyNamedWithDifferentSuperClassCubit extends ProperlyNamedCubit {
   ProperlyNamedWithDifferentSuperClassCubit();
 }
 
-class ClassExtendingOtherCubitClassWithoutSuffix extends NotProperlyNamed {
+class /*[2*/ClassExtendingOtherCubitClassWithoutSuffix/*2]*/ extends NotProperlyNamed {
```

Comment:
What is the difference between this and `[!!]`? `[!!]` works if there is only one diagnostic?

Reply:
Yes, `[!` is an alias for `/*[0*/`. There's also `^`, which is an alias for the position marker `/*0*/`.